lib/ansible/modules/network/libreswitch/ops_command.py

```diff
@@ -126,7 +126,7 @@
 """
 import traceback
 
-import ansible.module_utils.openswitch
+import ansible.module_utils.libreswitch
```

Comment:
too keep compatibility for those using the old name of the lib, this should be a try/except Import with fallback to the old name.